application/src/main/kotlin/com/studentcenter/weave/application/meetingTeam/service/domain/impl/MeetingTeamDomainServiceImpl.kt

```diff
@@ -113,6 +122,19 @@ class MeetingTeamDomainServiceImpl(
         meetingTeamRepository.deleteById(id)
     }
 
+    @Transactional
+    override fun publishById(id: UUID): MeetingTeam {
+        val meetingTeam = meetingTeamRepository.getById(id)
+        return if(meetingTeam.isPublished()) {
+            meetingTeam
+        } else {
+            createMeetingTeamMemberSummary(meetingTeam)
+                .also { meetingTeamMemberSummaryRepository.save(it) }
+                .let { meetingTeam.publish() }
+                .also { meetingTeamRepository.save(it) }
```

Comment:
p4) also 안에서 다 처리하거나 Run으로 따로 뺴는 것도 가독성이 괜찮아보여요!
```suggestion
                .also { meetingTeamMemberSummaryRepository.save(it) }
                .run {
                    meetingTeamRepository.save(meetingTeam.publish())
                }
```

Reply:
내부적으로 객체 전환이 있어서 run으로 처리하자니 반환타입이 Unit이라 애매하고, let으로 하나로 퉁치기에는
```kotlin
            createMeetingTeamMemberSummary(meetingTeam)
                .let {
                    meetingTeamMemberSummaryRepository.save(it)
                    val publishedMeetingTeam = meetingTeam.publish()
                    meetingTeamRepository.save(publishedMeetingTeam)
                    publishedMeetingTeam
                }
```
이런식으로 한줄 더들어가서 보기 불편하더라구요! 🤔